core/tests/blockchain_repair_test.go

```diff
@@ -39,7 +39,7 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/ethereum/go-ethereum/miner"
+	"github.com/ethereum/go-ethereum/miner" //nolint:typecheck
```

Comment:
why typecheck?

Reply:
It was giving lint error earlier. Now it's not. Strange :(

Reply:
Anyways, will remove it.